spec/coverage/.resultset.json

```diff
@@ -0,0 +1,7 @@
+{
+  "RSpec": {
+    "coverage": {
+    },
+    "timestamp": 1443180861
+  }
+}
```

Comment:
you probably want to add `spec/coverage/*` to your `.gitignore` These files should not be version controlled.